### PR TITLE
Including samczsun feedback

### PR DESCRIPTION
**Pull request by wh01s7**

### CODE REVIEW

File: 2.0/0x100-General/0x108-G8-Denial-of-Service.md [hunk 1]
```diff
@@ -21,7 +21,8 @@ Category “G8” lists requirements related to the possible denial of service o
 | **G8.8** | Verify that there are no costly operations in a loop. | 
 | **G8.9** | Verify that there are no calls to untrusted contracts in a loop. | 
 | **G8.10** | Verify that if there is a possibility of suspending the operation of the contract, it is also possible to resume it. | 
-| **G8.11** | Verify that if allow lists and list deny are used, it does not interfere with normal operation of the system | 
+| **G8.11** | Verify that if allow lists and list deny are used, it does not interfere with normal operation of the system. | 
+| **G8.12** | Verify that there are no DoS caused by overflows and underflows. |
```
Comment (damianrusinek) [inline 2.0/0x100-General/0x108-G8-Denial-of-Service.md]: are -> is

File: 2.0/0x100-General/0x106-G6-Communications.md [hunk 2]
```diff
@@ -19,7 +19,7 @@ Category “G6” lists requirements related to the function calls between the v
 | **G6.3** | Verify that delegatecall is not used with untrusted contracts. | 
 | **G6.4** | Verify that third party contracts do not shadow special functions (e.g. revert). | 
 | **G6.5** | Verify that the contract does not, check whether the address is a contract using *extcodesize* opcode. | 
-| **G6.6** | Verify that re-entrancy attack is mitigated by blocking recursive calls from other contracts and following Check-Effects-Interactions pattern. Do not use *call* and *send* functions unless it is a must. | 
+| **G6.6** | Verify that re-entrancy attack is mitigated by blocking recursive calls from other contracts and following Check-Effects-Interactions pattern. Do not use *send* function unless it is a must. |
```
Comment (damianrusinek) [inline 2.0/0x100-General/0x106-G6-Communications.md:22]: If we accept `call`, why can't we accept `send`?

File: 2.0/0x200-Components/0x203-C3-Oracle.md [hunk 3]
```diff
@@ -25,4 +25,6 @@ Category “C3” lists requirements related to the Oracle smart contract as one
 
 For more information, see also:
 
-* [The Dangers of Price Oracles in Smart Contracts](https://www.youtube.com/watch?v=YGO7nzpXCeA)
\ No newline at end of file
+* [ORACLES](https://ethereum.org/en/developers/docs/oracles/)
+* [The Dangers of Price Oracles in Smart Contracts](https://www.youtube.com/watch?v=YGO7nzpXCeA)
+* [So you want to use a price oracle](https://samczsun.com/so-you-want-to-use-a-price-oracle/)
```
Comment (damianrusinek) [inline 2.0/0x200-Components/0x203-C3-Oracle.md:30]: This should go to Interactions (with oracles) I guess.

**Review by damianrusinek — CHANGES_REQUESTED** [review]

File: 2.0/0x200-Components/0x203-C3-Oracle.md (hunk 3 above)
Comment (wh01s7) [reply 2.0/0x200-Components/0x203-C3-Oracle.md:30]: I assume that to design a secure oracle, we need to be aware of the risks of integration. I am happy to add a guide on building secure oracle as soon as I find one :D

File: 2.0/0x100-General/0x108-G8-Denial-of-Service.md (hunk 1 above)
Comment (wh01s7) [reply 2.0/0x100-General/0x108-G8-Denial-of-Service.md]: Thanks

File: 2.0/0x100-General/0x106-G6-Communications.md (hunk 2 above)
Comment (wh01s7) [reply 2.0/0x100-General/0x106-G6-Communications.md:22]: "Any smart contract that uses transfer() or send() is taking a hard dependency on gas costs by forwarding a fixed amount of gas: 2300."  https://consensys.net/diligence/blog/2019/09/stop-using-soliditys-transfer-now/

**Review by damianrusinek — APPROVED** [review]

**Pull request merged**